code/modules/clothing/suits/armor.dm

```diff
@@ -43,6 +43,12 @@
 	item_state = "blueshift"
 	custom_premium_price = 600
 
+/obj/item/clothing/suit/armor/vest/corporate
+	name = "Corporate Jacket"
```

Comment:
this is a jacket, but only has the area protection of a vest, and also has the armor of a vest? lower the armor and make it cover arms